CI/tests/test_pvc_scenario.sh

```
@@ -0,0 +1,31 @@
+set -xo pipefail
+
+source CI/tests/common.sh
+
+command="docker"
+#command=$(command_to_run)
+#echo "final $command"
+#if [ "$command" == "" ]
+#then
+#  echo "please install podman or docker before continuing"
+#  exit 1
+#fi
+
+function functional_test_pvc_scenario {  
+  export PVC_NAME=kraken-test-pvc
+  export POD_NAME=kraken-test-pod
```

Need to add the creation of the kraken-test-pod and kraken-test-pvc; right now getting below errror: 

```2021-12-02 17:53:19,588 [ERROR] Failed to run kubectl describe pvc <pvc_name> -n <namespace> | grep -E 'Mounted By:|Used By:' | grep -Eo '[^: ]*$', error: Command 'kubectl describe pvc <pvc_name> -n <namespace> | grep -E 'Mounted By:|Used By:' | grep -Eo '[^: ]*$'' returned non-zero exit status 1.```

---

Hi, I'm not quite sure how the CI are supposed to work, how they are executed... For that test to work we need to deploy a PV, PVC and pod. To do so, we can use the yaml definition I created on kraken repo: https://github.com/cloud-bulldozer/kraken/blob/master/CI/scenarios/volume_scenario.yaml

---

Yes we can use the same one, I think we should just be able to do the following at the bottom of this file

```
kubectl create -f https://raw.githubusercontent.com/cloud-bulldozer/kraken/master/CI/scenarios/volume_scenario.yaml
functional_test_pvc_scenario
kubectl delete -f https://raw.githubusercontent.com/cloud-bulldozer/kraken/master/CI/scenarios/volume_scenario.yaml
```

I realized the error message is separate from having the pod and stuff created. So ignore the error code I gave

---

added, I tested only this script locally, it works as expected